aptly/server/mirrors.sls

```diff
@@ -87,6 +87,18 @@ aptly_{{ mirror_name }}_mirror:
     - file: aptly_wrapper
   {%- endif %}
 
+aptly_{{ mirror_name }}_mirror_edit:
+  cmd.run:
+  - name: aptly mirror edit {% if mirror.get('udebs', False) %}-with-udebs=true {% endif %}{% if mirror.get('sources', False) %}-with-sources=true {% endif %}{% if mirror.get('filter') %}-filter="{{ mirror.filter }}" {% endif %}{% if mirror.get('filterdeps') %}-filter-with-deps {% endif %}-architectures={{ mirror.architectures }} {{ mirror_name }}
```

Comment:
You should add onlyif or unless option to avoid editing mirror when not needed.

Reply:
@fpytloun I've now added a onlyif clause to avoid editing mirror when not needed.